cgp/genome.py

```diff
@@ -7,6 +7,13 @@
 from .node import Node, OperatorNode
 from .primitives import Primitives
 
+try:
```

Comment:
if we only need this in one function, you could delay the import such that users without sympy only get an error when they call the function that needs it.

also, i don't see where the `sympy_available` variables are used, also, i think they are set the wrong way around? `ModuleNotFoundError` means sympy is *not* available, right?

in case we don't want to rely on sympy, maybe we can misuse the `compute_key_from_numpy...` function from `utils.py` to check that the output of the graph and the desired target function are *functionally* identical?